Gemfile

```diff
@@ -11,3 +11,4 @@ end
 gem 'jruby-openssl', :platform=> :jruby
 gem 'activemodel', ENV['RAILS_VERSION'] if ENV['RAILS_VERSION']
 gem 'linkeddata', '~> 1.99'
+gem 'rake', '< 12'
```

Comment:
@dchandekstark Chris informs me that he already ran transpec on AF 8.  Is there some reason you are using AF 7?